src/components/App.js

```diff
@@ -0,0 +1,92 @@
+import React, { useState, useEffect } from 'react';
+import './App.css';
+import axios from "axios";
+import Pokemon from "./pokemons/Pokemon";
+import pokeball from "../assets/pokeball.gif";
+
+function App() {
+    const [pokemons, setPokemons] = useState([]);
+    const [error, setError] = useState(false);
+    const [currentUrl, setCurrentUrl] = useState(`https://pokeapi.co/api/v2/pokemon/`);
+    const [nextUrl, setNextUrl] = useState(null);
+    const [previousUrl, setPreviousUrl] = useState(null);
+    const [loading, setLoading] = useState(true);
+
+    // console.log("NextURL?", nextUrl);
+    // console.log("CurrentURL", currentUrl);
+    // console.log("PreviousURL", previousUrl);
+
+    const showNextPage = () => {
+        setCurrentUrl(nextUrl)
+    }
+
+    const showPreviousPage = () => {
+        setCurrentUrl(previousUrl)
+    }
+
+    useEffect(() => {
+        setTimeout(() => setLoading(false), 2000)
+    }, [])
```

Comment:
Ik weet niet zo goed waar je dit voor hebt gedaan. Je zou in de `fetchPokemon` functie aan het begin loading op true kunnen zetten en na afloop weer op false